test/e2e/kubectl.go

```diff
@@ -138,6 +138,28 @@ func readTestFileOrDie(file string) []byte {
 	return framework.ReadOrDie(path.Join(kubeCtlManifestPath, file))
 }
 
+const kubectlRetryStr = "the object has been modified; please apply your changes to the latest version and try again"
```

Comment:
This error is from `pkg/registry/generic/registry/store.go`. Consider `const` the error string there?